src/storage/IndexRepresentationStore.ts

```diff
@@ -65,12 +65,18 @@ export class IndexRepresentationStore extends PassthroughStore {
   }
 
   /**
-   * Makes sure the stored media range matches the highest weight preference.
+   * Makes sure the stored media range explicitly matches the highest weight preference.
```

Comment:
With the current solution, if the range of this class is set to `text/*` and the accept header is about `text/html` (or reversed) there will still be a match. Does this count as explicit matching as well?

Reply:
> Does this count as explicit matching as well?

Indeed, and we test for that case.